### DISCUSSION TITLE
Auto import doesn't work for some new components

### DISCUSSION THREAD

**Discussion by anchan42** [primefaces/community#1787] [PrimeVue] [answered]
It doesn't work for new components like iconfield, inputicon and floatlabel?
I guess the resolver just need updating.
How do I do that?

**Comment by Muhammad-Anas785** [comment]
if you got the answer please also tell me

**Reply by anchan42** [reply]
Using the latest version 0.27 works. Toast component still need to be imported though.

**Comment by tugcekucukoglu** [comment, marked as answer]
You could try to use version 0.27.0. https://github.com/unplugin/unplugin-vue-components/releases/tag/v0.27.0

**Reply by anchan42** [reply]
Thanks:)
